README.md

```diff
@@ -12,12 +12,15 @@ Deployments resources offer two strategies to deploy changes: `RollingUpdate` an
 ## Documentation
 To learn more about Argo Rollouts go to the [complete documentation](https://argoproj.github.io/argo-rollouts/).
 
+> Do not install argo-rollouts from the master branch install.yaml.  This manifest uses the `latest` tag for the image which may be unstable.  Make sure to use the `stable` branch for all installations unless testing the latest iteration.
```

Comment:
We shouldn't need to call this out. In place of this comment, I think the GitHub README.md should have a Quickstart section (the same to the documentation), like:

## Quickstart

Argo Rollouts can be installed by running the following commands:
```
$ kubectl create namespace argo-rollouts
$ kubectl apply -n argo-rollouts -f https://raw.githubusercontent.com/argoproj/argo-rollouts/stable/manifests/install.yaml
```

Reply:
Cool